src/components/main.jsx

```diff
@@ -122,6 +127,22 @@ export default class Main extends React.Component {
     if (this.state.tool === 'measure') {
       this.setState({ lengthMeasurePoints: [ ...this.state.lengthMeasurePoints, { lat, lon } ] });
     }
+    if(this.state.tool == 'route-planner'){
+      const pointType = this.state.routePlannerPickMode
+      let newRoutePlannerPoints = null
+
+      if(pointType == 'start')
+        newRoutePlannerPoints = update(this.state.routePlannerPoints, {
+          start: { lat: {$set: lat.toFixed(6) }, lon: {$set: lon.toFixed(6) }}
+        });
+      else if(pointType == 'finish')
+        newRoutePlannerPoints = update(this.state.routePlannerPoints, {
+          finish: { lat: {$set: lat.toFixed(6) }, lon: {$set: lon.toFixed(6) }} // TODO: how to use key name (start/finish) from variable here to avoid repeating the same code?
```

Comment:
odstranim to. ja som skusal este dva ine router APIs (mapbox a mapquest, ale obe vracali podivne trasy) a pri debugovani mapboxu som skusal obmedzit pocet desatinnych miest.